### PR TITLE
Bbd about page test

### PR DESCRIPTION
**Pull request by Abdulaziz-rgb**

### CODE REVIEW

File: API/Tests/BaseTest.cs [hunk 1]
```diff
@@ -12,15 +12,12 @@ namespace API.Tests
...
         protected APIUtils.API api;
-        protected Client client;
         protected ICustomLogger logger;
-
         protected static ApiResourcesModel ApiResourcesData;
         
         [SetUp]
         public void SetUp()
         {
-            client = Client.Instance;
```
Comment (evgkas) [inline API/Tests/BaseTest.cs]: Client.Instance must be initialized at least once. API.cs using this instance to get the response. It's not really clear but this line should be here or you may add null check for Client.GetClient()

**Review by evgkas — CHANGES_REQUESTED** [review]

File: API/Tests/BaseTest.cs (hunk 1 above)
Comment (Abdulaziz-rgb) [reply API/Tests/BaseTest.cs]: Ok noted Yauheni Thanx

**Comment by NikitaGavruk** [comment]
Resolve please conflicts

**Comment by NikitaGavruk** [comment]
And remove API fixes from this PR and move it to standalone

**Pull request merged**